package.json

```
@@ -36,37 +36,36 @@
   },
   "exports": {
     ".": {
-      "import": "./lib/index.mjs",
-      "default": "./lib/index.js"
+      "import": "./dist/lib/index.mjs",
+      "default": "./dist/lib/index.js"
     },
-    "./cli/": "./cli/",
-    "./generators/": "./generators/",
-    "./jdl/": "./jdl/",
-    "./lib/": "./lib/",
-    "./utils/": "./utils/",
+    "./cli/": "./dist/cli/",
+    "./generators/": "./dist/generators/",
+    "./jdl/": "./dist/jdl/",
+    "./lib/": "./dist/lib/",
+    "./utils/": "./dist/utils/",
     "./package.json": "./package.json",
     "./esm/cli": "./cli/index.mjs",
     "./esm/generators/*": "./generators/*/esm.mjs",
     "./esm/generators": "./lib/constants/generators.mjs",
     "./esm/priorities": "./lib/constants/priorities.mjs",
     "./esm/support": "./lib/support/index.cjs"
```

Is there a reason for not update?

```suggestion
    "./esm/cli": "./dist/cli/index.mjs",
    "./esm/generators/*": "./dist/generators/*/esm.mjs",
    "./esm/generators": "./dist/lib/constants/generators.mjs",
    "./esm/priorities": "./dist/lib/constants/priorities.mjs",
    "./esm/support": "./dist/lib/support/index.cjs"
```

---

Yes, I explained it in the commit: https://github.com/jhipster/generator-jhipster/pull/19559/commits/5263183b9cb32368b6e306aaa44a72af6785919c